feature/trainee/signup/src/main/java/co/kr/tnt/trainee/signup/TraineeProfileSetupPage.kt

```diff
@@ -63,8 +70,7 @@ fun TraineeProfileSetupScreen() {
     )
 
     Scaffold(
-        // TODO 버튼 클릭 시 트레이너/트레이니 화면으로 이동
-        topBar = { TnTTopBarWithBackButton(onBackClick = {}) },
+        topBar = { TnTTopBarWithBackButton(onBackClick = { onBackClick() }) },
```

Comment:
```suggestion
        topBar = { TnTTopBarWithBackButton(onBackClick = onBackClick) },
```

람다도 리컴포지션 카운트에 영향을 줍니다!

Reply:
전부 수정했습니다!